storage/pebble/bootstrap.go

```diff
@@ -45,21 +48,24 @@ func NewRegisterBootstrap(
 		// key detected, attempt to run bootstrap on corrupt or already bootstrapped data
 		return nil, ErrAlreadyBootstrapped
 	}
+
 	checkpointDir, checkpointFileName := filepath.Split(checkpointFile)
 	return &RegisterBootstrap{
-		checkpointDir:      checkpointDir,
-		checkpointFileName: checkpointFileName,
 		log:                log.With().Str("module", "register_bootstrap").Logger(),
 		db:                 db,
+		checkpointDir:      checkpointDir,
+		checkpointFileName: checkpointFileName,
 		leafNodeChan:       make(chan *wal.LeafNode, checkpointLeafNodeBufSize),
 		rootHeight:         rootHeight,
+		registerCount:      atomic.NewUint64(0),
 	}, nil
 }
 
 func (b *RegisterBootstrap) batchIndexRegisters(leafNodes []*wal.LeafNode) error {
-	b.log.Debug().Int("batch_size", len(leafNodes)).Msg("indexing batch of leaf nodes")
 	batch := b.db.NewBatch()
 	defer batch.Close()
+
+	b.log.Trace().Int("batch_size", len(leafNodes)).Msg("indexing batch of leaf nodes")
```

Comment:
I believe this is to monitor the progress. 

Actually, `OpenAndReadLeafNodesFromCheckpointV6` is already doing that, since it's logging the percentage of reading the trie, and is blocked if database is not storing the registers fast enough, which means if the log shows the reading percentage is going up, it means leaf nodes are being indexed properly.

Reply:
yea, exactly. the checkpoint loader also logs based on % progress, resulting in much fewer messages